backend/apps/auth/src/utils/jwt.py

```
@@ -0,0 +1,104 @@
+        페이로드 (Dict[str, Any])
+    """
+    try:
+        payload = jwt.decode(token, JWT_SECRET, algorithms=[JWT_ALGORITHM])
+
+        if payload.get("type") != "refresh":
+            raise HTTPException(401, detail="Invalid token")
+
+        return payload["user_id"]
+        
+    except JWTError as e:
+        raise HTTPException(401, detail=f"Invalid token: {str(e)}")
```

Ruff (0.14.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)